### PR TITLE
Replace dev topic

### PR DESCRIPTION
**Pull request by danielcopaciu**

### CODE REVIEW

**Comment by sbuliarca** [comment]
Seems like CODEOWNERS didn't work :)

File: dev-aws/finance/invoice-payment-events.tf [hunk 1]
```diff
@@ -1,9 +1,5 @@
 resource "kafka_topic" "invoice_fulfillment" {
-  name               = "invoice.payment.events"
+  name               = "dd-run.events"
   replication_factor = 3
   partitions         = 10
-  config = {
```
Comment (sbuliarca) [inline dev-aws/finance/invoice-payment-events.tf:5]: Do you wanna go with the defaults?

Comment (danielcopaciu) [reply dev-aws/finance/invoice-payment-events.tf:5]: Yes

**Review by sbuliarca — APPROVED** [review]

**Comment by danielcopaciu** [comment]
@sbuliarca any idea why CODEOWNERS failed?

**Comment by sbuliarca** [comment]
> @sbuliarca any idea why CODEOWNERS failed?

maybe we need to target more exclusively rather than * for pusub: https://github.com/utilitywarehouse/kafka-cluster-config/blob/main/CODEOWNERS#L2

**Pull request merged**